GoogleSignIn/Sources/Public/GoogleSignIn/GIDSignIn.h

```diff
@@ -93,7 +90,7 @@ typedef void (^GIDDisconnectCallback)(NSError *_Nullable error);
 ///
 /// @param callback The `GIDSignInCallback` block that is called on completion.  This block will be
```

Comment:
I think you'll need to update this comment (and any other that uses `GIDSignInCallback`).

Reply:
Done.